sdk/test/NetStandard/UnitTests/Core/AWSSDKUtilsTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Amazon.Util;
+using System.Text;
+using Xunit;
+
+namespace UnitTests.NetStandard.Core
+{
+    [Trait("Category", "Core")]
+    public class AWSSDKUtilsTests
```

Comment:
Since the `#else` block in the `ToHex` method is really just for .NET Framework can you copy the tests to the .NET Framework unit tests in the `sdk\test\UnitTests\Custom` folder. Hopefully as part of V4 we can do some test restructure cleanup so we don't have to copy the test between the .NET Framework and .NET Standard+.